package/SUSEConnect.changes

```diff
@@ -3,6 +3,7 @@ Sun Jul 21 10:00:00 UTC 2021 - Natnael Getahun <ngetahun@suse.com>
 
 - Update to 0.3.31
 - Disallow registering via SUSEConnect if the system is managed by SUSE Manager.
+- Add subscription name to output of 'SUSEConnect --status'
```

Comment:
I thought this release was accepted.

Reply:
Ok, I didn't know it's already released. I can add a new changelog version.